lib/form.js

```diff
@@ -37,10 +37,7 @@ module.exports = function(req, opts){
   opts.qs = opts.qs || qs;
 
   // raw-body returns a Promise when no callback is specified
-  return Promise.resolve()
-    .then(function() {
-      return raw(inflate(req), opts);
-    })
+  return raw(inflate(req), opts)
```

Comment:
I think why we write this code is becaulse `inflate` will throw in some situation, so we need to wrap it in a promise chain.